### DISCUSSION TITLE
Get currently pinned message when joining chatroom?

### DISCUSSION THREAD

**Discussion by bananacity** [zSoulweaver/kient#26] [Q&A] [answered]
Just wondering what's the code I need to call to get the currently pinned message in chat when joining a chatroom?

When inspecting the network tab it's in this get request response for kick "https://kick.com/api/v2/channels/1227772/messages" which returns the currently pinned message and some messages from chat (don't need these) not sure if it's implemented in this library or not?

**Comment by zSoulweaver** [comment, marked as answer]
There are new API routes to do this specifically, but they don't seem to be working yet, so the above endpoint the the one you need. I've just released a new version that adds the above endpoint.

**Reply by bananacity** [reply]
Awesome, thanks!
